src/common/SurgeStorage.cpp

```diff
@@ -2269,6 +2269,36 @@ bool SurgeStorage::resetToCurrentScaleAndMapping()
     return true;
 }
 
+void SurgeStorage::loadTuningFromSCL(const fs::path &p)
+{
+    try
+    {
+        retuneToScale(Tunings::readSCLFile({p.u8string()}));
```

Comment:
I don't think you need the braced initialization here (and below).

Reply:
Will fix.